### ISSUE TITLE
Transition effects

### ISSUE THREAD

**Issue by zakkak** [dannyedel/dspdfviewer#19]
Hello,

Thanks for this great job.

I was wondering what are your thoughts about supporting some transition effects in the dspdfviewer.

**Comment by dannyedel** [comment]
First of all, sorry for making you wait so long. I had some computer issues and was unable to access github : / 

Regarding the transition effects: In my mind, any kind of effect/delay goes against my initial idea of having an instant reply to a keypress (hence the whole pre-rendering thing) and I personally have not missed it. However, I am not the only user of the software anymore and if there is a wish in the userbase for that, I don't see any problem with integrating this. [I just think I'm not able to program this myself, since I have no knowledge of graphics rendering. All I do is replace a pixmap].

We'd have to talk about how to integrate a code like that, but I think its reasonable to demand the following things:
  * We need an example latex presenation for testing that includes transition commands
  * The transition code will probably need some kind of high-function graphics library. I know Qt has some OpenGL integration and pretty much every graphics card these days implements OpenGL. However, if the code doesn't yet depend (through Qt-poppler) on OpenGL, I'd appreciate a compile-time switch to turn the functionality (and thus dependency) off.
  * There should be a runtime switch (commandline or such) to ignore transition commands, just in case someone wants instant-render even if the PDF says otherwise.

tl;dr:

Sounds cool, I cannot program it myself, but I will happily integrate it.

**Comment by zakkak** [comment]
Unfortunately, I cannot program it myself either.  I can help with the first bullet though. 

Let's see if anyone else can help with this :)
